tests/transport_test.go

```diff
@@ -77,8 +77,7 @@ var _ = Describe("Transport Tests", func() {
 		importer, err := utils.FindPodByPrefix(c, ns, common.ImporterPodName, common.CDILabelSelector)
 		Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Unable to get importer pod %q", ns+"/"+common.ImporterPodName))
 
-		err = utils.WaitTimeoutForPodStatus(c, importer.Name, importer.Namespace, v1.PodRunning, 5*utils.PodWaitForTime)
-		Expect(err).NotTo(HaveOccurred(), "Operation timeout on importer pod %q", ns+"/"+common.ImporterPodName)
+		err = utils.WaitTimeoutForPodStatus(c, importer.Name, importer.Namespace, v1.PodRunning, utils.PodWaitForTime)
```

Comment:
We are not checking the 'err' variable anywhere, why are we setting it? It will fail the go report card.

Reply:
oh, it was simply there before :) I can remove it while at it..

Reply:
Well my point is, that you are removing the Expect(err).NotTo(HaveOccured()) for no apparent reason, besides to not check err. Any reason you did that?

Reply:
The tests were sporadically failing after adding it :/ So I've just undo it for now till figuring it out.

Reply:
So I ran into the same issue, the problem is that in the negative tests, the status would not be wait we would expect all the time (due to retries and other things) so I have fixed it in my k8s and os version update patch.